src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,15 @@
-export const UserInfo = () => {};
+import React, { memo } from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
+
+type Props = {
+  user: User,
+};
+
+export const UserInfo: React.FC<Props> = memo(({ user }) => {
+  return (
```

Comment:
```suggestion
  return (
```
if the component has no body - return is redundant
or destructure your props before return